Sources/EpoxyBars/BarInstaller/BarWrapperView.swift

```diff
@@ -212,10 +215,18 @@ public final class BarWrapperView: UIView {
     }
   }
 
-  private func makeView(from model: InternalBarModeling, animated: Bool) -> UIView {
+  private func makeView(
+    from model: InternalBarModeling,
+    originalModel: InternalBarModeling?,
+    animated: Bool)
+  -> UIView
```

Comment:
nit: I believe this should be indented
```suggestion
    -> UIView
```